opentimelineio/adapters/fcp_xml.py

```diff
@@ -412,6 +422,10 @@ def _parse_top_level_track(track_e, kind, rate, element_map):
     if not track_items:
         return track
 
+    trackname_e = track_e.find('./name')
```

Comment:
Yeah, I noticed the <name> tag didn't refer to track in the link you posted. 
I agree with you concern, but find it strange that track names are omitted from the official XML export in the first place.
I'll let you guys make the decision. I'm happy to kill that "darling" if it's for the greater good :)

Reply:
I'm ok with this change, actually.  As long as FCP gracefully reads XMLs with the 'name' field I think its fine to include support for it.  Furthermore, in the EDL adapter we added a 'Flavor' switch to distinguish between EDLs (Avid and NuCoda), so I think that pattern is ok for this in the long run.  If we find its a problem as we scale up, we can disable behavior and fall back on lowest common denominator.

Since our ambition in the limit is that tools directly support OTIO rather than use OTIO to translate for interchange, we're trying to tease out these kinds of things that should be represented in the schema.